### DISCUSSION TITLE
SDXL model loading requires too much RAM (32GB+ maybe)

### DISCUSSION THREAD

**Discussion by okolenmi** [Comfy-Org/ComfyUI#1521] [Q&A] [answered]
I have a very simple SDXL workflow without even a refiner. It fails on first step - model loading. This step is not requiring VRAM (also there is no signs of using VRAM in task manager)

![изображение](https://github.com/comfyanonymous/ComfyUI/assets/55319814/4425640d-1c65-49e4-97f4-68fbfd77e7ec)

Not enough memory error.
It looks like 19GB of free memory (5GB were used before model loading) is not enough to load 6.8GB model. 

![изображение](https://github.com/comfyanonymous/ComfyUI/assets/55319814/a3899b63-414e-413d-8c29-e00bfe4128df)

Is this behaviour expected at the moment or not? ([model, that was used](https://civitai.com/models/142150/nutmeg-mix))

**Comment by ltdrdata** [comment]
Is it RAM or VRAM?

**Reply by okolenmi** [reply]
It's RAM. VRAM doesn't used during model loading as I can see.

**Comment by usigg3** [comment, marked as answer]
Make sure you have a windows page or swapfile.

**Reply by okolenmi** [reply]
It works with 8192MB page file.  It requires only 10GB of free RAM after model loading. I think, it can be optimized in some way. But, at least, it works.

**Reply by GothWave** [reply]
please, what is a windows page and swapfile? how can i search for this? (google only shows "face swap")
update: alright, i managed to understand what is a paging file size. My question now is, i need to allocate in the disk that is running ComfyUI or in the entire system?
